files/stop_times_ft.md

```diff
@@ -17,6 +17,8 @@ File MAY contain the following attributes:
 
 Optional Attributes		| Description										
 ----------				| -------------		
+`actual_arrival_time`	| Actual arrival time at a specific stop for a specific trip on a route in HH:MM:SS format measured from midnight.  For trips that span multiple dates, the time should be entered as a value greater than 2400000
```

Comment:
wonder if this needs to be a separate file with similar values for matching with dates/times as you have in routes_stats?